README_PAYLOAD.md

```diff
@@ -116,3 +118,30 @@ map{"key(1)": value(1),"key(2)": value(2),...,"key(n)": value(n)}
       "EOL":true //  <msgpack.raw>, <msgpack.boolean>
  }
 ```
+
+## short-trackle-message-packet
+
+This trackle message is a shortened version of an original trackle-message-packet. The original payload (device status,
+temperatures and configuration) is being replaced, by a hash over all fields (but the signature obviously). This hash
+is in the original trackle-message-packet used to calculate the signature. 
+
+This new shortened trackle message format allows us to process the trackle message in the Trust Service without
+having to handle the unhashed payload. 
+
+```
++=========+======+==================+======+=========+-------------+
+| VERSION | UUID |  PREV-SIGNATURE  | TYPE | HASH |  SIGNATURE     |
++=========+======+==================+======+=========+-------------+
+```
+VERSION: 0010|0011 => 2|3 (signed message with chained signatures)  https://github.com/ubirch/ubirch-protocol/blob/master/README.md#field-types
+UUID: hwDevice from the original trackle-message-packet
+PREV-SIGNATURE: the previous signature from the original trackle-message-packet
+TYPE: 0x56 (86)  https://github.com/ubirch/ubirch-protocol/blob/master/README.md#payload-type
+HASH: SHA-512 hash over the original trackle-message-packet fields: version, uuid, prev-signature, type and hash.
+SIGNATURE: the signature of the original trackle-message-packet 
+
+trackle message payload:
+```
+  [ ???
```

Comment:
@gruenwaldi What msgPack header should the payload field have? Could you edit this part?